db/db_impl.cc

```diff
@@ -4759,18 +4759,18 @@ Status DBImpl::WriteImpl(const WriteOptions& write_options,
       if (!cfd->mem()->IsEmpty()) {
         // We only consider active mem table, hoping immutable memtable is
         // already in the process of flushing.
-        size_t cfd_size = cfd->mem()->ApproximateMemoryUsage();
-        if (largest_cfd == nullptr || cfd_size > largest_cfd_size) {
-          largest_cfd = cfd;
-          largest_cfd_size = cfd_size;
+        uint64_t seq = cfd->mem()->GetLatestSeqWhenCreate();
```

Comment:
GetLatestSeqWhenCreate() effectively tells the time of last flush in this memtable, right? That's probably not what we want: in our use case most memtables are empty most of the time, and the time of last flush in a CF seems quite irrelevant. We want to flush the oldest unflushed data; the oldest unflushed piece of data lives in memtable with the smallest GetFirstSequenceNumber(), right?